scaleway/provider_test.go

```diff
@@ -19,3 +37,80 @@ func init() {
 }
 
 func testAccPreCheck(_ *testing.T) {}
+
+// getTestFilePath returns a valid filename path based on the go test name and suffix. (Take care of non fs friendly char)
+func getTestFilePath(t *testing.T, suffix string) string {
```

Comment:
All the testdata golden and cassettes will ALL be in the same folder. Plus all the file name will start with `test-acc-scaleway` that will not bring any information and might hurt the visibility in IDE pans.
Should we rename this pattern to something more practical? I think we can also rename the convention of TestAccScaleway because I don't see the point of it.